### PR TITLE
Update docusaurus v3

### LINKED ISSUE #265: Get rid of docusaurus-plugin-sass
Convert our files to regular CSS and use that instead

### PR DESCRIPTION
**Pull request by TheZoker**
- Update to [docusaurus v3](https://docusaurus.io/blog/releases/3.0)
- Remove SCSS plugin (closes #265)

### CODE REVIEW

File: src/css/global.css [hunk 1]
```diff
@@ -0,0 +1,96 @@
+/* stylelint-disable docusaurus/copyright-header */
+/**
+* Any CSS included here will be global. The classic template
+* bundles Infima by default. Infima is a CSS framework designed to
+* work well for content-centric websites.
+*/
+/* You can override the default Infima variables here. */
```
Comment (sdirix) [inline src/css/global.css:7]: Was this CSS copied from somewhere? Please link to the original source, especially if the source requires attribution per license

**Review by sdirix — COMMENTED** [review]
Looks good! I just have a question regarding the CSS

File: src/css/global.css (hunk 1 above)
Comment (TheZoker) [reply src/css/global.css:7]: This part of the code came from the auto-generated .scss file from docusaurus

**Review by sdirix — APPROVED** [review]
LGTM, Thanks!

**Pull request merged**